### PR TITLE
removed print and added logging

### PR DESCRIPTION
**Pull request by MohJumper**
Edit: replacing print statements in the rsu manager rest services with the logging framework for a more robust controller over logging.

### CODE REVIEW

File: GCP_cloud_functions/rsu-manager/rsu-proxies/rsu-rest-proxy/main.py [hunk 1]
```diff
@@ -35,25 +39,25 @@ class RSUCredSchema(Schema):
...
     schema = RSUCredSchema()
     errors = schema.validate(request_json["creds"])
     if errors:
-      print(errors)
+      logging.exception(errors)
       return str(errors), 400
   # If they don't exist and they should, return a failed HTTP response
   elif rsu_commands[request_json["command"]]["ssh_req"]:
-    print("Credentials required but not provided")
+    logging.info("Credentials required but not provided")
```
Comment (payneBrandon) [inline GCP_cloud_functions/rsu-manager/rsu-proxies/rsu-rest-proxy/main.py:60]: Let's set this to error level

File: GCP_cloud_functions/rsu-manager/rsu-proxies/rsu-rest-proxy/main.py [hunk 2]
```diff
@@ -2,6 +2,24 @@
...
+logging_levels = {
+        'debug': logging.DEBUG,
+        'info': logging.INFO,
+        'warning': logging.WARNING,
+        'error': logging.ERROR,
+        'crittical': logging.CRITICAL
+    }
+
+# return none if it does not exist
+env_log_level = os.getenv("LOGING_LEVEL")
```
Comment (payneBrandon) [inline GCP_cloud_functions/rsu-manager/rsu-proxies/rsu-rest-proxy/main.py:16]: We'll want to have a corresponding `import os` statement above to allow interactions with that library.

File: GCP_cloud_functions/rsu-manager/rsu-count-emailer/main.py [hunk 3]
```diff
@@ -3,6 +3,25 @@
 from sendgrid import SendGridAPIClient
 from sendgrid.helpers.mail import Mail
 import os
+import logging
+
+
+logging_levels = {
```
[the remark this answers is not in the document]
Comment (payneBrandon) [reply GCP_cloud_functions/rsu-manager/rsu-count-emailer/main.py:9]: We can apply the same pattern for other setups below.

File: GCP_cloud_functions/rsu-manager/rsu-count-emailer/test-count-emailer.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import unittest
```
Comment (payneBrandon) [inline GCP_cloud_functions/rsu-manager/rsu-count-emailer/test-count-emailer.py:1]: Should this class be in a separate PR/branch?

**Review by payneBrandon — CHANGES_REQUESTED** [review]
A few comments, but getting closer!

File: GCP_cloud_functions/rsu-manager/rsu-count-emailer/main.py [hunk 5]
```diff
@@ -3,6 +3,10 @@
 from sendgrid import SendGridAPIClient
 from sendgrid.helpers.mail import Mail
 import os
+import logging
+
+# Create custom config logging
+logging.basicConfig(format='%(levelname)s:%(message)s', level=logging.INFO)
```
[the remark this answers is not in the document]
Comment (MohJumper) [reply GCP_cloud_functions/rsu-manager/rsu-count-emailer/main.py]: I did add checks to the environment level and then default to INFO if we can not find it.

File: GCP_cloud_functions/rsu-manager/rsu-count-emailer/test-count-emailer.py (hunk 4 above)
Comment (MohJumper) [reply GCP_cloud_functions/rsu-manager/rsu-count-emailer/test-count-emailer.py:1]: I moved this class to a different branch.

File: GCP_cloud_functions/rsu-manager/rsu-count-emailer/main.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (MohJumper) [reply GCP_cloud_functions/rsu-manager/rsu-count-emailer/main.py:9]: I changed all to the suggested code.

File: GCP_cloud_functions/rsu-manager/rsu-count-emailer/main.py [hunk 6]
```diff
@@ -5,22 +5,15 @@
 import os
 import logging
 
-
-logging_levels = {
+LOGGING_LEVEL = {
```
Comment (payneBrandon) [inline GCP_cloud_functions/rsu-manager/rsu-count-emailer/main.py:8]: We can remove all these "LOGGING_LEVEL" dictionary definitions now that we aren't using them as well.

**Review by payneBrandon — CHANGES_REQUESTED** [review]
Just the one remaining issue. Otherwise looks good!

File: GCP_cloud_functions/rsu-manager/rsu-count-emailer/main.py (hunk 6 above)
Comment (MohJumper) [reply GCP_cloud_functions/rsu-manager/rsu-count-emailer/main.py]: Right! I removed them now.

**Review by payneBrandon — APPROVED** [review]
Looks good, thanks!

**Pull request merged**